src/hooks/useScripture.tsx

```diff
@@ -232,7 +232,7 @@ export function useScripture({ // hook for fetching scripture
   async function fetchBook(fetchParams, ignoreSha = null) {
     try {
       const fetchLink = `${fetchParams?.resourceLink}/${fetchParams?.reference?.projectId}`
-      const shaToBeIgnored = ignoreSha && ignoreSha === resourceState?.sha // we will reload if ignoreSha given and it matches current sha
+      const shaToBeIgnored = ignoreSha === null || ignoreSha === resourceState?.sha // we will reload if ignoreSha given and it matches current sha
```

Comment:
If you are untangling a logic bug, I think there needs to be some renaming of variables.  In my mind if ignoreSha is null, then sha is not to be ignored.